sbt-dotty/sbt-test/source-dependencies/export-clauses/test

```diff
@@ -0,0 +1,22 @@
+> compile
+$ copy-file changes/B1.scala B.scala
+> compile
+$ copy-file changes/C1.scala C.scala
+> compile
+$ copy-file changes/D1.scala D.scala
+> compile
+# This should be 7 because of:
```

Comment:
To make this more robust and easier to debug, I suggest adding a `checkIterations` call after every `compile`, that way we detect issues earlier.

Reply:
done